XLS-0045-prepublish-validator-lists/README.md

```diff
@@ -0,0 +1,824 @@
+<pre>
+title: <b>Prepublish Validator Lists</b>
+type: released in rippled 1.7.0
+description: Future activation date and improved expiration handling of UNLs
+author: <a href="mailto:ed@ripple.com">Ed Hennis (Ripple)</a>
+</pre>
+
+
+## Abstract
+
+Adds a future effective date to UNLs so that all nodes in the peer network can switch to a new UNL simultaneously. 
+Creates a new version of the VL file spec to allow publishing multiple UNLs in one file, including current and
+future UNLs. Adds enforcement of the UNL expiration date so that rippled will act similarly to being amendment
+blocked if it has no active valid UNL.
+
+:bangbang: Note that these changes were merged into rippled in
+    [1.7.0-b10](https://github.com/XRPLF/rippled/pull/3720) on
+    January 11, 2021. The original pull request was titled
+    ["Support UNLs with future effective dates: (UNL v2)"](https://github.com/XRPLF/rippled/pull/3619). The
+    original specification was published on a (now?) private repository. The purpose of this XLS is to make
+    the specification of these changes more accessible so that related tooling can be updated, and to make it
+    easier for UNL publishers to use the new format. Any improvements to this spec can be integrated into
+    rippled in a later pull request. Most of this content is written from the perspective of rippled before
+    these changes were merged.
```

Comment:
```suggestion
This spec was finalized and implemented in `rippled` version
[1.7.0-b10](https://github.com/XRPLF/rippled/pull/3720) on January 11, 2021. The
original pull request was titled ["Support UNLs with future effective dates:
(UNL v2)"](https://github.com/XRPLF/rippled/pull/3619). The purpose of this XLS
is to make the specification of these changes widely accessible so that related
tooling can be updated. Any further improvements to the VL format should be
defined in a new spec and implemented in `rippled` with a new pull request. Most
of the following content is written from the perspective of `rippled` before
these changes were merged.
```